src/main/java/com/rackspace/jenkins_nodepool/Utils.java

```diff
@@ -0,0 +1,4 @@
+package com.rackspace.jenkins_nodepool;
+
+public class Utils {
```

Comment:
Whats this?

Reply:
Odd, the implementation and test didn't get pushed in this changeset.  I have a full implementation with tests for strip leading zeros and random string.  My mistake.  Will remove.

Reply:
Removed